ls1intum/Artemis
# `Development`: Add documentation for text exercises

### Checklist
#### General



### Motivation and Context


Currently, there is no documentation for text exercises.

### Description

This PR adds basic documentation for text exercises.

### Steps for Testing

Check out the new documentation here: https://artemis-platform--6651.org.readthedocs.build/en/6651/user/exercises/textual.html


### Review Progress






#### Code Review
